### PR TITLE
35 fix batch

### PR DESCRIPTION
**Pull request by celina-ayl**

### CODE REVIEW

File: hub/examples/image_retraining/retrain.py [hunk 1]
```diff
@@ -745,7 +744,43 @@ def add_final_training_ops(class_count, final_tensor_name, bottleneck_tensor):
...
+
+            # t=1.0 (training)  → uses live batch statistics
+            # t=0.0 (inference) → uses learned running averages
+            t = tf.cast(is_training, tf.float32)
+            mean_to_use = t * batch_mean + (1.0 - t) * bn_mov_mean
+            var_to_use  = t * batch_var  + (1.0 - t) * bn_mov_var
+
+            logits = tf.nn.batch_normalization(
+                pre_activations, mean_to_use, var_to_use, bn_beta, bn_gamma, epsilon)
+
```
Comment (renney-11) [inline hub/examples/image_retraining/retrain.py:783]: This manual reimplementation of batch normalization isn't needed. tf.keras.layers.BatchNormalization already handles all of this internally (the gamma, beta, moving mean, moving variance, epsilon, and the training/inference switch). 

Only change that was required from the original: replace training=True with training=is_training. The entire block from bn_gamma down to tf.nn.batch_normalization(...) should be able to be replaced with something like:

`logits = tf.keras.layers.BatchNormalization()(
    pre_activations,
    training=is_training
)`

**Review by renney-11 — CHANGES_REQUESTED** [review]
The is_training placeholder you added and the update_ops / control_dependencies block are both correct, nice! Just simplify the BN call

**Review by renney-11 — APPROVED** [review]
